components/Button/Button.tsx

```diff
@@ -20,13 +20,13 @@ const Button: FC<Props> = ({
   className,
 }) => {
   const styles = {
-    primary: '', // @todo: add styles when using these button types in ResourcesCards
+    primary: 'bg-primary-purple hover:bg-primary-purple/80 text-white rounded-[10px] duration-500', // @todo: add styles when using these button types in ResourcesCards
```

Comment:
We can remove the comment now.

```suggestion
    primary: 'bg-primary-purple hover:bg-primary-purple/80 text-white rounded-[10px] duration-500',
```